docs/userguide.rst

```diff
@@ -69,3 +79,7 @@ Python reference
 .. autofunction:: rhui_aliases
```

Comment:
Would you mind dropping these from the docs since we are deprecating them so we're only presenting the API that we want people to use going forward?